tests/modules/site_settings/resources/test_file_operations.py

```diff
@@ -53,10 +53,7 @@ def test_file_settings(admin_user, flask_app_client, flask_app, db, request, tes
         site_setting = resp.json
         assert site_setting['key'] == 'footer_image'
 
-        # this is to test that GET /api/v1/site-settings/file is *truly* only getting file-type values
```

Comment:
Can no longer set arbitrary houston keys, they must be in HOUSTON SETTINGS but there are plenty of them so this isn't necessary